test/test-notification-model-updates.js

```diff
@@ -519,6 +534,24 @@ test('Test notifications', function(t) {
     });
   });
 
+  t.test('check change stream', function(tt) {
+    var change = changeStream.read();
+    tt.equal(change.target, 'ServiceInstance-1');
+    tt.equal(change.type, 'create');
+    change = changeStream.read();
+    tt.equal(change.target, 'ServiceInstance-1');
+    tt.equal(change.type, 'update');
+    change = changeStream.read();
+    var eventCnt = 0;
+    while (change !== null) {
```

Comment:
Is this being done for debug reasons? I expected something more like:
```js
while (changeStream.read() !== null) {
  eventCnt++;
}
```

I'm mostly wondering if I'm missing something..